package.json

```diff
@@ -6,7 +6,7 @@
   "main": "dist/testTrack.bundle.js",
   "module": "dist/testTrack.js",
   "scripts": {
-    "build": "rollup -c",
+    "build": "tsc && rollup -c",
```

Comment:
`rollup` won't fail the build if TypeScript fails to compile, so running the compiler before will gives us warnings in CI about broken types.